kyaml/openapi/openapi.go

```diff
@@ -372,13 +403,20 @@ const (
 	// kubernetesGVKExtensionKey is the key to lookup the kubernetes group version kind extension
 	// -- the extension is an array of objects containing a gvk
 	kubernetesGVKExtensionKey = "x-kubernetes-group-version-kind"
+
 	// kubernetesMergeKeyExtensionKey is the key to lookup the kubernetes merge key extension
 	// -- the extension is a string
 	kubernetesMergeKeyExtensionKey = "x-kubernetes-patch-merge-key"
+
 	// kubernetesPatchStrategyExtensionKey is the key to lookup the kubernetes patch strategy
 	// extension -- the extension is a string
 	kubernetesPatchStrategyExtensionKey = "x-kubernetes-patch-strategy"
 
+
+	// kubernetesMergeKeyMapList is the list of merge keys when there needs to be multiple
+	// -- the extension is an array of strings
+	kubernetesMergeKeyMapList = "x-kubernetes-list-map-keys"
```

Comment:
Who maintains/updates this extension in the openapi swagger.json ? And how ?

Reply:
That's the job of the apimachinery sig.

Using it here is the recommendation of @apelisse from #3125, a pr that was closed as draft.

We should probably come to a conclusion here, and not start a new pr.

Reply:
>> Who maintains/updates this extension in the openapi swagger.json ? And how ?
> That's the job of the apimachinery sig.

Partly. We maintain the machinery, people writing Kubernetes types (from various sigs) and CRDs do use them.